sklearn/ensemble/_forest.py

```diff
@@ -174,6 +178,40 @@ def _parallel_build_trees(tree, forest, X, y, sample_weight, tree_idx, n_trees,
     return tree
 
 
+def _permutation_importances_oob(
+    estimator,
+    X,
+    y,
+    sample_weight,
+    n_samples,
+    n_samples_bootstrap,
+    random_state,
+):
+    """Compute the feature permutation importance given a tree."""
+    # avoid circular dependence
+    from ..inspection import permutation_importance
+
+    unsampled_indices = _generate_unsampled_indices(
+        estimator.random_state,
+        n_samples,
+        n_samples_bootstrap,
+    )
+
+    if sample_weight is None:
+        sample_weight = np.ones(n_samples)
+
+    return permutation_importance(
+        estimator,
+        X[unsampled_indices, :],
+        y[unsampled_indices],
+        scoring=None,
+        n_repeats=1,
+        n_jobs=1,
+        random_state=random_state,
+        sample_weight=sample_weight[unsampled_indices],
+    ).importances[:, 0]
```

Comment:
I do not know `permutation_importance` sufficiently, but at first sight, expliciting why `n_jobs=1` is used would helps: IIUC as `_permutation_importances_oob` is called within `joblib.Parallel.__apply__` and as `permutation_importance` also calls it, we explicitly force sequential execution by setting `n_jobs=1`.

Also, why does one only takes `.importances[:, 0]`? Does `permutation_importance` compute more than what we need here?